projects/angular-calendar/src/modules/common/calendar-event-actions.component.ts

```diff
@@ -15,8 +16,14 @@ import { CalendarEvent, EventAction } from 'calendar-utils';
           href="javascript:;"
           *ngFor="let action of event.actions; trackBy: trackByActionId"
           (mwlClick)="action.onClick({ event: event })"
+          (mwlKeydown)="action.onClick({ event: event })"
           [ngClass]="action.cssClass"
           [innerHtml]="action.label"
+          tabindex="0"
+          role="button"
+          attr.aria-label="{{
+            { label: action.label } | calendarA11y: 'actionButtonLabel'
+          }}"
```

Comment:
Just a minor style nit, can this use the the square brackets instead of interpolation?

`[attr.aria-label]="{ label: action.label } | calendarA11y: 'actionButtonLabel'"`